ts/Series/PackedBubble/PackedBubblePoint.ts

```diff
@@ -147,6 +148,15 @@ class PackedBubblePoint extends BubblePoint implements DragNodesPoint {
         }
     }
 
+    public setState(
+        state?: (StatesOptionsKey|''),
```

Comment:
Is the `|''` part superfluous? Seem likes the StatesOptionsKey type should already include it